### DISCUSSION TITLE
Use own main template

### DISCUSSION THREAD

**Discussion by c-gross** [orchidsoftware/platform#2174] [General]
Hi, is there a way to overwrite the main app.blade.php template?

I would like to change a css class (container-md to container-fluid) to get the full screen with for the content.

Edit the config/plattform.php did not work:

```
'template' => [
        'app' => 'app',
        'header' => 'brand.header',
        'footer' => 'brand.footer',
    ],
```

**Comment by tabuna** [comment]
You can use template overriding, you can read about it here: https://orchid.software/en/docs/configuration/#overriding-blade-templates
